host/loadelf.c

```diff
@@ -718,6 +727,12 @@ static oe_result_t _sgx_load_enclave_properties(
     result = OE_OK;
 
 done:
+    if (result != OE_OK)
+        OE_TRACE_ERROR(
```

Comment:
Since you already invoked OE_TRACE_ERRORs before jumping to done, you don't need the extra check and print here.

Reply:
removed